lib/cylc/cfgspec/suite.py

```diff
@@ -352,6 +354,16 @@ def coerce_interval_list(value, keys, args, back_comp_unit_factor=1,
                 'execution timeout'           : vdr( vtype='interval_minutes'),
                 'reset timer'                 : vdr( vtype='boolean', default=False ),
                 },
+            'events' : {
+                'handler'                     : vdr( vtype='string_list', default=[] ),
+                'handler events'              : vdr( vtype='string_list', default=[] ),
+                'handler retry delays'       : vdr( vtype='interval_minutes_list', default=[] ),
+                'mail events'                 : vdr( vtype='string_list', default=[] ),
+                'mail from'                   : vdr( vtype='string' ),
+                'mail retry delays'           : vdr( vtype='interval_minutes_list', default=[] ),
+                'mail smtp'                   : vdr( vtype='string' ),
+                'mail to'                     : vdr( vtype='string' ),
+                },
```

Comment:
This is like the original cylc event hooks config - one handler and list of events for it - which we changed because it doesn't allow different handlers for different events.  Is this intended to replace the current system (just left in for back compat) or do you want to keep both?